spec/PeriodicBackgroundSync-index.bs

```diff
@@ -0,0 +1,75 @@
+<pre class='metadata'>
+Title: Web Periodic Background Synchronization
+Status: CG-DRAFT
+ED: https://wicg.github.io/BackgroundSync/spec/PeriodicBackgroundSync-index.html
+Shortname: periodic-background-sync
+Level: 1
+Editor: Mugdha Lakhani, Google, nator@chromium.org
+Abstract: This specification describes a method that enables web applications to periodically synchronize data and content in the background.
+Group: wicg
+Repository: WICG/BackgroundSync
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:dfn; for:/; text:browsing context
+spec:service-workers; type:dfn; text:frame type
+</pre>
+
+Introduction {#intro}
+=====================
+
+  <em>This section is non-normative.</em>
+
+  Web Applications often run in environments with unreliable networks (e.g., mobile phones) and unknown lifetimes (the browser might be killed or the user might navigate away).
+  This makes it difficult for web apps to keep their content and state in sync with servers. This API is intended to reduce the time between content creation and content synchronization between the servers and the web app.
+
+  As this API relies on service workers, functionality provided by this API is only available in a [=secure context=].
+
+  ## Example ## {#example}
+  Requesting a [=periodic Background Sync opportunity=] at a mininimum interval of one day from a [=browsing context=]:
+
+  <pre class="lang-js">
+    async function registerPeriodicNewsCheck() {
+      const registration = await navigator.serviceWorker.ready;
+      try {
+        await registration.periodicSync.register('fetch-news', {
+          minInterval: 24 * 60 * 60 * 1000,
+        });
+      } catch (error) {
+        console.log('Periodic Sync could not be registered!');
+      }
+    }
+  </pre>
+
+  Reacting to a <code>periodicsync</code> event within a [=service worker=]:
+
+  <pre class="lang-js">
+    self.addEventListener('periodicsync', event => {
+      event.waitUntil(fetchAndCacheLatestNews());
+    });
+  </pre>
+
+  In the above example <code>fetchAndCacheLatestNews</code> is a developer-defined function.
```

Comment:
While the name is descriptive, it would help here to add some more colour:

> ...is a developer-defined function that fetches the latest news articles from a server and stores them locally, for example using the Cache API, for offline consumption.

Reply:
done